tools/essence-feature-usage-stats/requirements.txt

```diff
@@ -0,0 +1,19 @@
+black==23.10.1
```

Comment:
we should ask dependabot to monitor these as well

Reply:
done